.gitignore

```diff
@@ -4,5 +4,7 @@
 .env
 .DS_Store
 venv/
+.venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
Consider whether you want to ignore `db.sqlite3`. If this file is part of your development environment and not needed in production, it's fine to ignore it. However, if it's a critical part of your application, you might want to reconsider ignoring it.